### PR TITLE
Home Lab Blog Post

### PR DESCRIPTION
**Pull request by ssmiller25**

### CODE REVIEW

File: _posts/2015-08-19-home-lab-rebuild.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+{% include JB/setup %}
+
+Recently a hard drive went out on my main home VM Server.  It was in a RAID, so
+there was no data loss.  But as I had to shut everything down to replace 
+hardware (no hot swap...), it makes me think about also maintaining the 
+software stack for the lab.
+
+Currently I have two physical systems that are libvirt/KVM hosts for my 
+virtual machines, including the main firewall and fileserver.  I've had my eye
+on a few newer technologies, so I'd make the lab flexible.
```
Comment (ghost) [inline _posts/2015-08-19-home-lab-rebuild.md]: This sentence feels awkward, not sure what you're trying to say. You'd like to make the lab flexible maybe?

Comment (ssmiller25) [inline _posts/2015-08-19-home-lab-rebuild.md]: I updated the language to hopefully make it more clear.  Let me know what
you think.

**Comment by skeetersim** [comment]
sounds good.

File: _posts/2015-08-19-home-lab-rebuild.md (hunk 1 above)
Comment (ghost) [inline _posts/2015-08-19-home-lab-rebuild.md]: sounds good

**Pull request merged**